boards/limifrog-v1/include/periph_conf.h

```diff
@@ -27,13 +27,13 @@ extern "C" {
  * @name Clock system configuration
  * @{
  **/
-#define CLOCK_HSE           (16000000U)         /* external oscillator */
+#define CLOCK_HSI           (16000000U)         /* external oscillator */
```

Comment:
The PLL was already using the HSI, I can change it back to HSE if it has en external XTAL